src/main/webapp/open_hours.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded", async function() {
+  if(document.getElementById("registration-form-area")) {
+  const mondayTimeOption = new TimeOption("Monday", true, null);
+  const tuesdayTimeOption = new TimeOption("Tuesday", true, null);
+  const wednesdayTimeOption = new TimeOption("Wednesday", true, null);
+  const thursdayTimeOption = new TimeOption("Thursday", true, null);
+  const fridayTimeOption = new TimeOption("Friday", true, null);
+  const saturdayTimeOption = new TimeOption("Saturday", true, null);
+  const sundayTimeOption = new TimeOption("Sunday", true, null);
+  }
+});
+
+class TimeOption {
+  constructor(day, forRegistration, organization) {
+    this.day = day;
+    this.forRegistration = forRegistration;
+    this.organization = organization;
+    this.optionArea = document.getElementById("hours-option-area");
+
+    this.dayOptionArea = document.createElement("div");
+    this.dayOptionArea.classList.add("day-option-area");
+    this.timeInputArea = document.createElement("div");
+
+    this.dayLabel = document.createElement("label");
+    this.dayLabel.textContent = `${this.day}: `;
```

Comment:
usually when I see space in a text context, that suggests to me that you are trying to manually space something that should be spaced via css padding. Please check how this renders without the spaces?